### PR TITLE
Update notebook images to include kserve instead of kfserving. Also update repo name. 

### PR DESCRIPTION
**Pull request by mbaijal**
**Description of your changes:**
Update notebook images to include kserve instead of kfserving. Also update repo name. 

Tests have passed

**Testing:**
- [ ] Unit tests pass
- [ ] e2e tests pass
- Details about new tests (If this PR adds a new feature)
- Details about any manual tests performed

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: components/notebook-dockerfiles/pytorch/requirements.txt [hunk 1]
```diff
@@ -14,7 +14,7 @@ ipywidgets==7.7.1
 # Kubeflow Related Packages
 kfp==1.8.13
 kfp-server-api==1.8.3
-kfserving==0.6.1
+kserve==0.9.0
```
Comment (mbaijal) [inline components/notebook-dockerfiles/pytorch/requirements.txt:17]: Also tried with kserve 0.8.0.2

Comment (surajkota) [reply components/notebook-dockerfiles/pytorch/requirements.txt:17]: so do we have both images?
Kubeflow 1.6 has KServe 0.8
https://github.com/kubeflow/manifests/tree/v1.6-branch#kubeflow-components-versions
release notes doesnt callout any breaking changes so should be ok

File: tests/e2e/tests/test_notebook_images.py [hunk 2]
```diff
@@ -23,11 +23,11 @@
 CUSTOM_RESOURCE_TEMPLATES_FOLDER = "./resources/custom-resource-templates"
 
 NOTEBOOK_IMAGES = [
-    "public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-scipy:v1.5.0",
-    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.9.1-gpu-py39-cu112-ubuntu20.04-e3-v1.2",
-    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-tensorflow:2.9.1-cpu-py39-ubuntu20.04-e3-v1.2",
-    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-pytorch:1.12.0-gpu-py38-cu116-ubuntu20.04-ec2",
-    "public.ecr.aws/c9e4w0g3/notebook-servers/jupyter-pytorch:1.12.0-cpu-py38-ubuntu20.04-ec2",
+    "kubeflownotebookswg/jupyter-scipy:v1.6.0",
+    "public.ecr.aws/kubeflow-on-aws/notebook-servers/jupyter-tensorflow:2.9.1-gpu-py39-cu112-ubuntu20.04-e3-v1.2-2022-09-20",
```
Comment (surajkota) [inline tests/e2e/tests/test_notebook_images.py:27]: need to update the helm charts as well

https://github.com/awslabs/kubeflow-manifests/blob/main/charts/apps/jupyter-web-app/templates/ConfigMap/jupyter-web-app-config-tgcbb2gdkd-kubeflow-ConfigMap.yaml#L14-L19

**Review by surajkota — CHANGES_REQUESTED** [review]
helm chart

File: components/notebook-dockerfiles/pytorch/requirements.txt (hunk 1 above)
Comment (mbaijal) [reply components/notebook-dockerfiles/pytorch/requirements.txt:17]: 0.8.0 needs the awscli and boto3 to be even older hence kept it at 0.9.0 for kserve

**Comment by surajkota** [comment]
Also, will update all refs as part of this https://github.com/awslabs/kubeflow-manifests/search?p=1&q=c9e4w0g3

File: tests/e2e/tests/test_notebook_images.py (hunk 2 above)
Comment (mbaijal) [reply tests/e2e/tests/test_notebook_images.py:27]: done

**Review by surajkota — APPROVED** [review]

**Comment by mbaijal** [comment]
Merging with one approval and pending final test run - will comment here once tests pass.

**Comment by mbaijal** [comment]
> Merging with one approval and pending final test run - will comment here once tests pass.

Tests Passed.

**Pull request merged**